bin/run-ci.sh

```diff
@@ -6,9 +6,12 @@ SCALA_VERSION="$1"
 sbt -Dfile.encoding=UTF-8 \
   -J-XX:ReservedCodeCacheSize=256M \
   -J-Xmx3046M -J-Xms3046M -J-server \
+  +mimaReportBinaryIssues \
   zincRoot/test:compile \
-  scalafmt::test \
-  test:scalafmt::test \
+# drop scalafmt on the 1.0.0 branch to dogfood 1.0.0-RC2 before there is a sbt 1.0 of new-sbt-scalafnt
```

Comment:
This doesn't work, comments make the script non-executable and `crossTestBridges` is not found. You have to move them out of the commands section.